### PR TITLE
Added fields to review application

### PR DESCRIPTION
**Pull request by ssh16**
# Review onboarding application

Added some fields to the review screen of the document provider onboarding application

## What

Adding some more content to the application 

## Why

So Island.is can review application of document providers

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: libs/application/templates/document-provider-onboarding/src/forms/ReviewApplication.ts [hunk 1]
```diff
@@ -26,10 +26,62 @@ export const ReviewApplication: Form = buildForm({
...
+            buildDividerField({ name: 'Upplýsingar um stofnun' }),
             buildTextField({
               id: 'applicant.name',
-              name: 'Nafn umsækjanda',
+              name: 'Nafn stofnunar',
+              disabled: true,
+            }),
+            buildTextField({
+              id: 'applicant.email',
+              name: 'Netfang ',
```
Comment (fjandakornid) [inline libs/application/templates/document-provider-onboarding/src/forms/ReviewApplication.ts:37]: mættir sækja þetta úr messages gaurunum... ætti að vera bara það sama og er Application megin

Comment (ssh16) [reply libs/application/templates/document-provider-onboarding/src/forms/ReviewApplication.ts:37]: auðvitað :D

Comment (hinriksteinar) [reply libs/application/templates/document-provider-onboarding/src/forms/ReviewApplication.ts]: Þetta er komið svo má resolve-a þetta :)

**Review by hinriksteinar — APPROVED** [review]
Mjög flott, sé ekkert athugavert 👍

**Pull request merged**